<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBookDialog</class>
 <widget class="QDialog" name="NewBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加新书</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>581</width>
     <height>471</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;等线 Light&quot;;</string>
   </property>
   <property name="title">
    <string>添加新书</string>
   </property>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>30</y>
      <width>151</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLineEdit" name="Edit_Title">
     <property name="placeholderText">
      <string>不需要输入书名号</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="Edit_Author"/>
    <widget class="QLineEdit" name="Edit_Publisher"/>
    <widget class="QComboBox" name="Edit_Department">
     <property name="currentText">
      <string>计算机学院</string>
     </property>
     <item>
      <property name="text">
       <string>计算机学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>理学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>马克思主义学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人文学院</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>信息与通信技术学院</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="Edit_Permission">
     <item>
      <property name="text">
       <string>任何人</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>教职工</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>学生</string>
      </property>
     </item>
    </widget>
    <widget class="QDateEdit" name="Edit_Indate">
     <property name="displayFormat">
      <string>yy.MM.dd</string>
     </property>
     <property name="date">
      <date>
       <year>2017</year>
       <month>3</month>
       <day>9</day>
      </date>
     </property>
    </widget>
    <widget class="QLineEdit" name="Edit_Amount"/>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;等线 Light&quot;;</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Label_Title">
       <property name="text">
        <string>书名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Author">
       <property name="text">
        <string>作者</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Publisher">
       <property name="text">
        <string>出版社</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Department">
       <property name="text">
        <string>图书部门</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>图书权限</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_InTime">
       <property name="text">
        <string>购入时间</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Label_Amount">
       <property name="text">
        <string>购入数量</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Button_save_next">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>200</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	background-image: url(:/background/resourse/button.png);
}
QPushButton:hover{
	background-image: url(:/background/resourse/button_h.png);
}
QPushButton:pressed{	
	background-image: url(:/background/resourse/button_P.png);
}
</string>
    </property>
    <property name="text">
     <string>保存并添加下一本</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_save_quit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>410</y>
      <width>120</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	background-image: url(:/background/resourse/button.png);
}
QPushButton:hover{
	background-image: url(:/background/resourse/button_h.png);
}
QPushButton:pressed{	
	background-image: url(:/background/resourse/button_P.png);
}
</string>
    </property>
    <property name="text">
     <string>保存并退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_cancel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>410</y>
      <width>120</width>
      <height>27</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	border:2px groove rgb(255,255,255,0,0);border-radius:10px;padding:2px 4px;
	font: 12pt &quot;Adobe 黑体 Std R&quot;;
color:white;
	background-image: url(:/background/resourse/button.png);
}
QPushButton:hover{
	background-image: url(:/background/resourse/button_h.png);
}
QPushButton:pressed{	
	background-image: url(:/background/resourse/button_P.png);
}
</string>
    </property>
    <property name="text">
     <string>取消并返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Intro">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线 Light</family>
      <pointsize>14</pointsize>
      <weight>3</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 14pt &quot;等线 Light&quot;;</string>
    </property>
    <property name="text">
     <string>图书简介</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Edit_Intro">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>60</y>
      <width>256</width>
      <height>211</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25 10pt &quot;等线 Light&quot;;
background-image: url(:/background/resourse/b_W.png);</string>
    </property>
    <property name="placeholderText">
     <string>100字以内</string>
    </property>
   </widget>
   <zorder>Button_save_next</zorder>
   <zorder>Button_save_quit</zorder>
   <zorder>Button_cancel</zorder>
   <zorder>Label_Intro</zorder>
   <zorder>Edit_Intro</zorder>
   <zorder>widget</zorder>
   <zorder>splitter</zorder>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/background/resourse/books7.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Button_cancel</sender>
   <signal>clicked()</signal>
   <receiver>NewBookDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>520</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>246</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
